frontend/src/component/releases/ReleaseManagement/ReleasePlanTemplateCard.tsx

```diff
@@ -71,8 +77,16 @@ export const ReleasePlanTemplateCard = ({
                     <StyledCreatedBy>
                         Created by {template.createdByUserId}
                     </StyledCreatedBy>
-                    <StyledMenu onClick={(e) => e.preventDefault()}>
-                        <ReleasePlanTemplateCardMenu template={template} />
+                    <StyledMenu
+                        onClick={(e) => {
+                            e.preventDefault();
+                            e.stopPropagation();
+                        }}
+                    >
+                        <ReleasePlanTemplateCardMenu
+                            template={template}
+                            clickHandler={clickHandler}
+                        />
```

Comment:
This one confuses me a bit. If we always want to navigate on click, why not wrap the card in a link instead?

Reply:
The card is already wrapped in a click, but I have to override that for the menu, and thus also specify an onclick for the menu items